BrowserKit/Sources/ToolbarKit/AddressToolbar.swift

```diff
@@ -0,0 +1,10 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+/// Protocol representing an address toolbar.
+public protocol AddressToolbar {
+    func configure(_ state: AddressToolbarState)
```

Comment:
nit? I like seeing the param name if the name of the function is short/doesn't tell what it needs as param
```suggestion
    func configure(state: AddressToolbarState)
```